scripts/common.py

```diff
@@ -23,6 +23,19 @@
 RELEASE_BRANCH_NAME_PREFIX = 'release-'
 
 
+def run_cmd(cmd_str):
+    """Runs the command and returns the output.
+    Raises subprocess.CalledProcessError upon failure.
+
+    Args:
+        cmd_str: str. The command string to execute
+
+    Returns:
+        str. The output of the command.
+    """
+    return subprocess.check_output(cmd_str.split(' ')).strip()
```

Comment:
I don't think you need this function, actually. Just pass in a list of "command tokens" from the get-go every time you call subprocess.check_output; that's pretty standard.

Reply:
Done.

Reply:
Just to confirm - you meant removing the splitting part by passing tokens as a list or removing the function overall? I have made changes only by removing the splitting part. If you meant removing the function, I will make changes according to that.

Reply:
Removing the function overall.